.github/workflows/readme-updated.yaml

```diff
@@ -0,0 +1,27 @@
+name: Check README.md helptext
```

Comment:
overall I might have not bothered with a dedicated CI for that.

Since you have it in pre-commit -- could have a pre-commit CI to check all of them if so was desired.

<details>
<summary>here is a number of projects calling to pre-commit in their workflows -- didn't check yet for best example</summary> 

```shell
❯ grep -l pre-commit */.github/workflows/*
aiohttp_retry/.github/workflows/python-package.yml
asv/.github/workflows/pre-commit.yml
belay/.github/workflows/pre-commit.yaml
black/.github/workflows/lint.yml
boto3/.github/workflows/lint.yml
briefcase/.github/workflows/ci.yml
briefcase/.github/workflows/pre-commit-update.yml
celery/.github/workflows/linter.yml
cibuildwheel/.github/workflows/test.yml
codecarbon/.github/workflows/pre-commit.yml
commitizen/.github/workflows/pythonpackage.yml
django-rest-framework/.github/workflows/pre-commit.yml
filesystem_spec/.github/workflows/main.yaml
gpt-engineer/.github/workflows/pre-commit.yaml
home-assistant/.github/workflows/ci.yaml
lark/.github/workflows/mypy.yml
lazy_loader/.github/workflows/lint.yml
mamba/.github/workflows/linters.yml
mdformat/.github/workflows/tests.yaml
mlflow/.github/workflows/autoformat.yml
mlflow/.github/workflows/lint.yml
mlflow/.github/workflows/recipe-template.yml
mypy/.github/workflows/test.yml
networkx/.github/workflows/lint.yml
nilearn/.github/workflows/update_precommit_hooks.yml
ome-zarr-py/.github/workflows/precommit.yml
pixi/.github/workflows/pre-commit.yml
prefect/.github/workflows/static-analysis.yaml
pydantic/.github/workflows/ci.yml
pydantic/.github/workflows/docs-update.yml
pyparsing/.github/workflows/pre-commit.yml
python-cookiecutter/.github/workflows/test.yml
requests/.github/workflows/lint.yml
ruff/.github/workflows/ci.yaml
ruff/.github/workflows/release.yaml
singularity-hpc/.github/workflows/main.yml
yarl/.github/workflows/reusable-linters.yml
zarr-python/.github/workflows/needs_release_notes.yml

```
</details>

Reply:
Ive filed a separate issue, since that would involve refactoring other CI jobs (codespell) https://github.com/con/duct/issues/68